### ISSUE TITLE
No module named 'torch._six'

### ISSUE THREAD

**Issue by Ishihara-Masabumi** [Wangt-CN/DisCo#50]
When I run the google colab DisCo_Demo.ipynb, the following error occurred.

```
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-25-7aa731641df4>](https://localhost:8080/#) in <cell line: 6>()
      4 
      5 from utils.wutils_ldm import *
----> 6 from agent import Agent_LDM, WarmupLinearLR, WarmupLinearConstantLR
      7 import torch
      8 from config import BasicArgs

3 frames
[/usr/local/lib/python3.10/dist-packages/deepspeed/runtime/utils.py](https://localhost:8080/#) in <module>
     16 
     17 import torch
---> 18 from torch._six import inf
     19 import torch.distributed as dist
     20 

ModuleNotFoundError: No module named 'torch._six'
```
But, 'torch._six' was on or under torch==1.7.0.
Next, the following error occurred.

```
!pip install pip install torch==1.7.0
Requirement already satisfied: pip in /usr/local/lib/python3.10/dist-packages (23.1.2)
Collecting install
  Downloading install-1.3.5-py3-none-any.whl (3.2 kB)
ERROR: Could not find a version that satisfies the requirement torch==1.7.0 (from versions: 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0, 2.0.1, 2.1.0)
ERROR: No matching distribution found for torch==1.7.0
```

Please revise the gooble colab torch version on or over 2.0.0.

**Comment by frogeyedpeas** [comment]
I might be running into similar issues: https://github.com/Wangt-CN/DisCo/issues/53 did you manage to fix this? also what operating system are you using?

**Comment by Wangt-CN** [comment]
Hi @Ishihara-Masabumi , thanks for the reporting. I have just tried to build a new python 3.8 env and run the installation. Everything is ok. Could you pls check if directly running the training/evaluation command can work? (not run the ipynb)

**Comment by Ishihara-Masabumi** [comment]
Does your (not run the ipynb) mean we can't run he google colab DisCo_Demo.ipynb?

**Comment by ljh0v0** [comment]
Hi, I solved this problem by using `deepspeed == 0.8.2`. Only version 0.8.2 works for this project. Lower versions has problem with `torch._six` and higher versions will lead to [this issue](https://github.com/Wangt-CN/DisCo/issues/33)
